### ISSUE TITLE
(Request) file imports or an IO module

### ISSUE THREAD

**Issue by CleanGlyph** [bendlang/bend#428]
**Is your feature request related to a problem? Please describe.**
A request. I don't see docs related to this, so forgive me if it's in the rust, and I'm not good enough at reading it. I want a way to load files into a datatype or even just raw that we can then build additional libraries/packages around it. 

I'm not sure where it is on your road map of features, probably a bit away, but it would let intermediate users jump in like myself! Either way, I'm stalking this language <3 

**Describe the solution you'd like**

Realistically though, just a simple IO library where I can write my own way of parsing csv, pickle, etc. 

In a perfect world I'd like a polars style of back end where it runs off the disk/memory, so if the IO library somehow made it easier for someone smarter than me to make a polar-bend library 👀 

**Describe alternatives you've considered**

I think having an IO module would be realistic, then let others worry about file types.

**Comment by developedby** [comment]
IO is coming very very soon, likely still this week (you can actually already use what's implemented, it's just not publicized to users).
Raw fille reads/writes are not in the initial version of the IO, only strings. But we'll soon add a way for people to add their own IO and FFI calls.

File imports are also coming soon, we're releasing an integrated package manager for bend and we'll be releasing some basic libraries to that, plus whatever users come up with.
Hopefully by the end of the week or next week.

**Comment by sergchernata** [comment]
Has there been any movement on this?

**Comment by developedby** [comment]
Yes, we have implemented basic file IO, you can read about it here https://github.com/HigherOrderCO/Bend/blob/main/docs/builtins.md and here https://github.com/HigherOrderCO/Bend/blob/main/src/fun/builtins.bend.
We're still working on consolidating the documentation in a nicer format.

You can impement your own IO as explained here https://github.com/HigherOrderCO/Bend/blob/main/docs/ffi.md, although it's not ssuper convenient to do ATM.

We have implemented imports, which you can read about here https://github.com/HigherOrderCO/Bend/blob/main/docs/imports.md and are finalizing our implementation of the package manager.

**Comment by VictorTaelin** [comment]
Bend is being reorganized. This repository (bendlang/bend) now belongs to Bend 2, a new implementation written from scratch, coming soon. Bend 1 lives on with its full history at https://github.com/HigherOrderCO/Bend1. We are closing every Bend 1 issue and pull request here: if this one still applies to Bend 1, please re-open it there.
